### PR TITLE
Backstop

### PR DESCRIPTION
**Pull request by mikemonteith**
## Add backstop tests to travis CI with docker

- Run backstop inside docker. Running `npm run backstop:test` now requires that docker is installed.
- Added a new Backstop stage to travis CI
- Updated baseline images

### CODE REVIEW

**Comment by AdamChrimes** [comment]
Looks good to me, @mikemonteith I've added the documentation if you wouldn't mind having a look. @mcheung-nhs would you be able to see if BackstopJS will run for you too?

File: package.json [hunk 1]
```diff
@@ -11,14 +11,15 @@
...
-    "backstop:clean": "rm -rf tests/backstop/backstop_data/bitmaps_test/*",
+    "backstop:ref": "backstop --config=tests/backstop/backstop.js reference --docker",
+    "backstop:test": "backstop --config=tests/backstop/backstop.js test --docker",
+    "backstop:clean": "rm -rf tests/backstop/bitmaps_test/*",
     "build-gh-pages": "gulp bundle && BASE_URL='/nhsuk-frontend' gulp docs:build",
     "build-gh-release": "gulp zip"
   },
   "dependencies": {
     "accessible-autocomplete": "^1.6.2",
+    "minimist": "^1.2.0",
```
Comment (AdamChrimes) [inline package.json]: @mikemonteith does this need to be a `devDependency`? As it will get installed for people who `npm install nhsuk-frontend`

Comment (mikemonteith) [reply package.json]: Yep! My bad.

**Comment by mcheung-nhs** [comment]
All running for me
👍

**Comment by AdamChrimes** [comment]
All good

**Pull request merged**